pkg/genericapiserver/config.go

```diff
@@ -71,6 +71,7 @@ type Config struct {
 	// allow downstream consumers to disable the index route
 	EnableIndex           bool
 	EnableProfiling       bool
+	EnableVersion         bool
```

Comment:
Why is this wrong?

How generic is the discovery client supposed to be? That's the unanswered question here IMO. If it is not generic, let's move the version route to pkg/routes.

For reference: origin has `/version/openshift` (https://github.com/openshift/origin/blob/a58b6d87f531075ce9b9d3caffca66c00dfcab59/pkg/cmd/server/origin/master.go#L349-L349). 

Is `/version/openshift` used as a replacement of kube's /version? What will a kubectl used against an origin cluster see and use?

It feels pretty strange to have